src/styles/ComponentPage.scss

```diff
@@ -59,26 +59,18 @@
   max-width: 100%;
   padding: 40px 16px 100px;
   transition: 200ms transform cubic-bezier(.4, 0, .2, 1) 50ms;
-  display: flex;
-  justify-content: center;
   width: 100%;
   overflow: auto;
+  display: flex;
```

Comment:
I think you can remove the drawer/variables import above

Reply:
Good catch. Removed.